Add EURC and XLM assets from Pendulum

gianfra-t opened the pull request:
Related issue: [Tasks/#434](https://github.com/pendulum-chain/tasks/issues/434)

green-ade committed:
Add LBTC

green-jay committed:
Merge branch 'master' into add-lbtc

gianfra-t committed:
add eurc and xlm, asset and config structure

nohaapav committed:
Fix: filter out virtual shares from spot most liquid route calc

nohaapav committed:
RELEASE: Releasing 1 package(s)

Releases:
  @galacticcouncil/sdk@6.1.2

gianfra-t committed:
adjust assets and fees

gianfra-t committed:
reorder assets alphabetically

nohaapav committed:
fix aave wrap & unwrap pool validations

nohaapav committed:
RELEASE: Releasing 1 package(s)

Releases:
  @galacticcouncil/sdk@6.1.3

nohaapav committed:
fix aave buy (flip)

nohaapav committed:
fix most liquid route calc for aave

nohaapav committed:
RELEASE: Releasing 1 package(s)

Releases:
  @galacticcouncil/sdk@6.1.4

nohaapav committed:
router sell all & dryrun

nohaapav committed:
RELEASE: Releasing 1 package(s)

Releases:
  @galacticcouncil/sdk@6.2.0

nohaapav committed:
bump sdk version

nohaapav committed:
RELEASE: Releasing 1 package(s)

Releases:
  @galacticcouncil/xcm-cfg@9.1.1

nohaapav committed:
fix failing tests

nohaapav committed:
regenerate lock

ebma commented on packages/xcm-cfg/src/configs/polkadot/hydration/index.ts:
```diff
@@ -287,7 +289,11 @@ const toLaos: AssetRoute[] = [toTransferTemplate(laos, laos_chain, 0.1)];
 const toEnergywebx: AssetRoute[] = [toTransferTemplate(ewt, energywebx, 0.02)];
 
-const toPendulum: AssetRoute[] = [toTransferTemplate(pen, pendulum, 1.1)];
+const toPendulum: AssetRoute[] = [
+  toTransferTemplate(pen, pendulum, 1.1),
+  toTransferTemplate(eurc, pendulum, 0.09),
+  toTransferTemplate(xlm, pendulum, 0.73),
+];
```
What's the meaning of these values? Are these coefficients that are applied to the base value of these assets to determine the costs?

gianfra-t replied:
These are the fees, in absolute values (units, not accounting for decimals). I didn't find in the docs though how or where are they used. The values are what I got when testing a live xcm transfer.

ebma replied:
Ah okay, understood.

ebma commented on packages/xcm-cfg/src/configs/polkadot/pendulum.ts:
```diff
@@ -23,6 +23,43 @@ const toHydration: AssetRoute[] = [
+  new AssetRoute({
+    source: {
+      asset: eurc,
+      balance: BalanceBuilder().substrate().system().account(),
```
Are you sure that this is correct? Only the native asset balance can be accessed in the `system::account()` query so I guess this is wrong?

gianfra-t replied:
You may be right, I didn't think about this and it wasn't important to create the transaction itself according to the tests, but it may be for the UI.

ebma replied:
Haha yeah I guess it's important, otherwise the balance won't show correctly.

ebma reviewed (commented):
Please link to the related ticket in the description of this PR.

nohaapav committed:
next: share token (MAX) & most liq route update
#119

nohaapav committed:
bump polkadot-api to latest

nohaapav committed:
publish new descriptors

nohaapav committed:
comment out dryRun descriptors (size)

nohaapav committed:
next: fix virtual share meta

nohaapav committed:
next: fix default pegs
#119

nohaapav committed:
next: whitelist router sell_all
#119

nohaapav committed:
next: readme WIP
#119

nohaapav committed:
Update README.md

nohaapav committed:
Update README.md

green-jay committed:
Merge branch 'master' into add-lbtc

Radovenchyk committed:
Update README.md

Maksandre committed:
chore: update Unique ws endpoint to geo load balancer

The previous WebSocket endpoint (wss://unique-rpc.dwellir.com) no longer works and may be deprecated by the Dwellir team. Updated to Unique Network's official geo load-balanced RPC

nohaapav committed:
Merge pull request #155 from Maksandre/update-unique-ws-endpoint

chore: update Unique ws endpoint to geo load balancer

nohaapav committed:
next: erc20 balance sub
#119

nohaapav committed:
next: sdk-swapp init
#119

nohaapav committed:
RELEASE: Releasing 1 package(s)

Releases:
  @galacticcouncil/xcm-cfg@9.1.2

nohaapav committed:
next: add token type
#119

nohaapav committed:
next: erc20 pool sub
#119

nohaapav committed:
next: fix test data

nohaapav committed:
next: update generic balance sub
#119

nohaapav committed:
next: fix asset types

nohaapav committed:
next: swapp example dapp
#119

nohaapav committed:
bump deps

nohaapav committed:
cleanup

nohaapav committed:
next: update readme
#119

nohaapav committed:
Merge pull request #154 from Radovenchyk/patch-1

chore: add license badge

nohaapav committed:
Update readme

nohaapav committed:
next: descriptor whitelist Router.Executed

nohaapav committed:
next: support aave
#119

nohaapav committed:
next: pairwise balance subscribe
#119

nohaapav committed:
next: update script section

nohaapav committed:
sdk: 7.x refactor & namespaces cleanup

nohaapav committed:
sdk: 7.x spec cleanup

green-jay committed:
Merge branch 'master' into add-lbtc

nohaapav committed:
Merge pull request #150 from galacticcouncil/add-lbtc

Add LBTC

nohaapav committed:
update doc

nohaapav committed:
sdk: h160 & erc20 utils

nohaapav committed:
sdk: swap supply & withdraw support

nohaapav committed:
sdk: evm client support

nohaapav committed:
sdk: buildWithdrawAndSellReserveTx support

nohaapav committed:
update xcm cfg hydration client

nohaapav committed:
cleanup consts

nohaapav committed:
RELEASE: Releasing 2 package(s)

Releases:
  @galacticcouncil/sdk@7.0.0
  @galacticcouncil/xcm-cfg@9.2.0

nohaapav committed:
sdk: fix export api module

nohaapav committed:
RELEASE: Releasing 1 package(s)

Releases:
  @galacticcouncil/sdk@7.0.1

nohaapav committed:
sdk: substrate transaction

nohaapav committed:
RELEASE: Releasing 1 package(s)

Releases:
  @galacticcouncil/sdk@7.0.2

gianfra-t committed:
replace balance method

gianfra-t committed:
Merge remote-tracking branch 'upstream/master' into add-eurc-xlm-support

ebma reviewed (approved):
I think it looks good to me. Feel free to _not merge_ this PR but create a new PR to the upstream repository with the changes on this branch.

gianfra-t committed:
add changeset

gianfra-t commented:
Thanks! [Here](https://github.com/galacticcouncil/sdk/pull/156#issuecomment-2862386365) is the PR to upstream. I think last step is to test once they provide the preview. And add the icons, but for that we need to confirm the id's are okay.
